src/backend/efiling-api/jag-efiling-api.yaml

```diff
@@ -82,6 +82,35 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/EfilingError"
+  "/submission/{id}/submit-filing-package":
```

Comment:
cant this just be:
```
/submission/{id}/filing-package
```
and since it is a post it is obvious that we are submitting the package?